starlite/config/app.py

```diff
@@ -47,7 +47,7 @@ class AppConfig(BaseModel):
     class Config(BaseConfig):
         arbitrary_types_allowed = True
 
-    after_exception: SingleOrList[AfterExceptionHookHandler]
+    after_exception: SingleOrList[AfterExceptionHookHandler] = []
     """An application level :class:`exception hook handler <starlite.types.AfterExceptionHookHandler>` or list thereof.
```

Comment:
these have also changed since you forked, e.g., `after_exception: List[AfterExceptionHookHandler]` is current typing for this.